src/posts/components/Post.tsx

```diff
@@ -0,0 +1,70 @@
+import "../../assets/styles/responsive.scss"; // تأكد من المسار الصحيح
+
+import React from "react";
+
+interface PostProps {
+  author: string;
+  authorImg: string;
+  img: string;
+  likes: number;
+  comments: number;
+  tags: string[];
+  content: string;
+}
+
+const Post: React.FC<PostProps> = ({
```

Comment:
suggesstion: you can simplify the declaration with: 
`const Post = ( { ... } : PostProps ) => {`